CodeEditModules/Modules/WorkspaceClient/src/Interface.swift

```diff
@@ -9,14 +9,19 @@ import Combine
 import Foundation
 
 public struct WorkspaceClient {
+
+	public var folderURL: () -> URL?
```

Comment:
honestly I wouldn't add this API. The `folderURL` is injected while initializing the client itself, so the consumer already has a reference of the directory somewhere

Reply:
`WorkspaceDocument` is the class responsible to build the `WorkspaceClient` 
https://github.com/CodeEditApp/CodeEdit/blob/01d95ba3752c2572a61a1057e71b0388441ccc1d/CodeEdit/Documents/WorkspaceDocument.swift#L95

Reply:
this means that `WorkspaceDocument` has a reference of the `folderURL`

Reply:
ok I added a reference `var folderURL: URL?` to `WorkspaceDocument`